Video.js

```diff
@@ -276,16 +276,20 @@ export default class Video extends Component {
     let uri = source.uri || '';
     if (uri && uri.match(/^\//)) {
       uri = `file://${uri}`;
-    } else if (uri === '') {
-      return null;
     }
 
     if (!uri) {
-      console.warn('Trying to load empty source.');
+      console.log('Trying to load empty source.');
     }
 
-    const isNetwork = !!(uri && uri.match(/^https?:/));
-    const isAsset = !!(uri && uri.match(/^(assets-library|ph|ipod-library|file|content|ms-appx|ms-appdata):/));
+    const isNetwork = !!(uri && uri.match(/^https?:/i));
+    const isAsset = !!(uri && uri.match(/^(assets-library|ph|ipod-library|file|content|ms-appx|ms-appdata):/i));
+
+    if (uri && !isNetwork && !isAsset) {
```

Comment:
If `uri` is `''` this will never run

change to

```typescript
if ((uri || uri == '') && !isNetwork && !isAsset) {
```

Reply:
This is done, thank you